.gitignore

```diff
@@ -13,27 +13,31 @@ build/
 !default.mode2v3
 *.perspectivev3
 !default.perspectivev3
-xcuserdata
+**/xcuserdata
 *.xccheckout
 *.moved-aside
-DerivedData
+*DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
 
 # Android/IntelliJ
 #
-build/
-.idea
-.gradle
-local.properties
+**/.idea
+**/.gradle
+**/local.properties
 *.iml
 
 # node.js
 #
-node_modules/
-npm-debug.log
-yarn-error.log
+**/node_modules
+**/npm-debug.log
+**/yarn-error.log
+**/package-lock.json
+yarn.lock*
```

Comment:
yarn.lock should not be ignored. Please remove it from the gitignore dir.

Reply:
Oki 😄